### PR TITLE
Inna kushchak

### PR DESCRIPTION
**Pull request by InnaKushchak**

### CODE REVIEW

File: src/test/java/InnaKushchakTest.java [hunk 1]
```diff
@@ -52,5 +54,34 @@ public void testContactUsButtonVerification() {
...
         Assert.assertEquals(getDriver().getTitle(), "Contact Us – AskOmDch");
     }
+    @Test
+    public void testWebDriver(){
+
+        WebElement featureProducts=getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
+        List<WebElement> products=featureProducts.findElements(By.tagName("li"));
+        for (WebElement elem:products) {
+         List<WebElement> x= elem.findElements(By.tagName("h2"));
+            System.out.println(x.size());
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:72]: System.out.println should not be in tests

File: src/test/java/InnaKushchakTest.java [hunk 2]
```diff
@@ -52,5 +54,34 @@ public void testContactUsButtonVerification() {
...
+        }
+    @Test
+    public void testWebDriverSearch(){
+
+        WebElement featureProducts=getDriver().findElement(By.xpath("//div[@class='wp-block-group alignfull']//ul"));
+        List<WebElement> products=featureProducts.findElements(By.tagName("li"));
+        products.get(0).getAttribute("baseURI");
+        products.get(0).findElement(By.tagName("h2")).getAttribute("textContent");
+        products.get(0).findElement(By.tagName("h2")).getAttribute("outerHTML");
+        System.out.println( products.get(0).findElement(By.tagName("h2")).getAttribute("innerHTML"));
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:93]: System.out.println should not be in tests

File: src/test/java/InnaKushchakTest.java [hunk 3]
```diff
@@ -52,5 +54,34 @@ public void testContactUsButtonVerification() {
...
+         List<WebElement> x= elem.findElements(By.tagName("h2"));
+            System.out.println(x.size());
+        }
+    }
+    @Test
+    public void testBestSellers(){
+            WebElement bestSellers=getDriver().findElement(By.xpath("//ul[@class='product_list_widget']"));
+            List<WebElement> validateProducts= bestSellers.findElements(By.tagName("li"));
+            for(WebElement listOfBest:validateProducts){
+
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:81]: Please add Assert to the test.

**Review by InnaKushchak — COMMENTED** [review]
ok

**Pull request closed without merging**